test/index.js

```diff
@@ -20,8 +20,10 @@ series([
   require('./basic')
   require('./multitarget')
 
-  if (process.platform !== 'win32') {
-    // Perform additional tests specific to building for OS X
+  // Perform additional tests specific to OS
+  if (process.platform === 'darwin') {
```

Comment:
`process.platform === 'darwin'` is not the same as `process.platform !== 'win32'`.